custom/icds_reports/sqldata/exports/growth_tracker_report.py

```diff
@@ -0,0 +1,178 @@
+                _check_case_presence(row['case_id'], 'lunch_count', data_past_month_2),
+                _check_case_presence(row['case_id'], 'lunch_count', data_past_month),
+                row['lunch_count'],
+                get_status(
+                    _check_case_presence(row['case_id'], current_month_stunting_column(self.beta),
+                                         data_past_month_2),
+                    'stunted',
+                    'Normal height for age',
+                    True
+                ),
+                get_status(
+                    _check_case_presence(row['case_id'], current_month_stunting_column(self.beta),
+                                         data_past_month),
+                    'stunted',
+                    'Normal height for age',
+                    True
+                ),
+                get_status(
+                    row[current_month_stunting_column(self.beta)],
+                    'stunted',
+                    'Normal height for age',
+                    True
+                ),
+                get_status(
+                    _check_case_presence(row['case_id'], current_month_wasting_column(self.beta),
+                                         data_past_month_2),
+                    'wasted',
+                    'Normal weight for height',
+                    True
+                ),
+                get_status(
+                    _check_case_presence(row['case_id'], current_month_wasting_column(self.beta), data_past_month),
+                    'wasted',
+                    'Normal weight for height',
+                    True
+                ),
+                get_status(
+                    row[current_month_wasting_column(self.beta)],
+                    'wasted',
+                    'Normal weight for height',
+                    True
+                ),
+                get_status(
+                    _check_case_presence(row['case_id'], 'current_month_nutrition_status', data_past_month_2),
+                    'underweight',
+                    'Normal weight for age',
+                    True
+                ),
+                get_status(
+                    _check_case_presence(row['case_id'], 'current_month_nutrition_status', data_past_month),
+                    'underweight',
+                    'Normal weight for age',
+                    True
+                ),
+                get_status(
+                    row['current_month_nutrition_status'],
+                    'underweight',
+                    'Normal weight for age',
+                    True
+                )
+            ]
+            excel_rows.append(row_data)
+        filters = [['Generated at', india_now()]]
+        if location:
+            locs = SQLLocation.objects.get(location_id=location).get_ancestors(include_self=True)
+            for loc in locs:
+                filters.append([loc.location_type.name.title(), loc.name])
+        else:
```

Comment:
Is this report available at National level?

Reply:
No